### ISSUE TITLE
Installing under Windows and Anaconda

### ISSUE THREAD

**Issue by dfkettle** [mido/mido#198]
Is there any way to install MIDO under Windows and Anaconda? I see MIDO listed in the default Anaconda repositories, but only for Linux and OSX.

Thanks.

**Comment by dunnousername** [comment]
I used ``pip install mido``, then ``pip install rtmidi-python`` (I think you need to do the second in an administrator anaconda powershell). I can't figure out how to get it to output properly, though (probably something on my part); if I figure it out and remember I'll post it here.

**Comment by dfkettle** [comment]
Thanks! Are you sure it was 'rtmidi-python' and not 'python-rtmidi'? I tried 'rtmidi-python' first and it installed with no errors reported, but it didn't work (I got an error on "import rtmidi"). However, looking at the rtmidi documentation, it refers to 'python-rtmidi'. So I uninstalled 'rtmidi-python' and installed 'python-rtmidi' (talk about confusing names!). Now it seems to work. At least I don't get an error or the import statement. See attached screen print.

![Screenshot 2019-08-04 07 31 04](https://user-images.githubusercontent.com/8409615/62423051-5fc50400-b68a-11e9-879d-985e5de1eac4.png)
